src/core/homepage_elements/journals/hooks.py

```diff
@@ -27,6 +27,6 @@ def yield_homepage_element_context(request, homepage_elements):
         else:
             featured_journals = request.press.featured_journals.all()
 
-        return {'featured_journals': featured_journals}
+        return {'featured_journals': sorted(featured_journals, key=lambda x: x.name)}
```

Comment:
This is what I meant when we spoke about this:
https://docs.python.org/3/library/operator.html#operator.attrgetter

The lambda is fine too though